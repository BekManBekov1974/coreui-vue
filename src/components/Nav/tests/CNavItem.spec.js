import { mount } from '@vue/test-utils'
import Component from '../CNavItem'

const ComponentName = 'CNavItem'
const defaultWrapper = mount(Component)
const customWrapper = mount(Component, {
  propsData: {

  },
  slots: {
    default: 'CNavItem text'
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(defaultWrapper.element).toMatchSnapshot()
  })
  it('renders correctly', () => {
    expect(customWrapper.element).toMatchSnapshot()
  })
})
