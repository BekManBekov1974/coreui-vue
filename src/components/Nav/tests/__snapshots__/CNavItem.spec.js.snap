// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CNavItem renders correctly 1`] = `
<li
  class="c-nav-item"
>
  <a
    class="c-nav-link"
    href="#"
    target="_self"
  />
</li>
`;

exports[`CNavItem renders correctly 2`] = `
<li
  class="c-nav-item"
>
  <a
    class="c-nav-link"
    href="#"
    target="_self"
  >
    CNavItem text
  </a>
</li>
`;
